src/views/Incorporation/IncorporationAgreement.vue

```diff
@@ -266,8 +264,7 @@ import { Getter } from 'pinia-class'
 import { useStore } from '@/store/store'
 import AgreementType from '@/components/common/AgreementType.vue'
 import { IncorporationAgreementIF } from '@/interfaces'
-import { RouteNames } from '@/enums'
-import { CorpTypeCd } from '@bcrs-shared-components/enums/'
```

Comment:
So this is weird. Whatever tool verifies types doesn't realize that CorpTypeCd from the shared repo is re-exported in our enums/index.ts file. They're the same enum but there was a lint warning.